client/src/hooks/useIncTypeOptions.js

```diff
@@ -17,42 +17,43 @@ export function useIncTypeOptions() {
 
   const [incTypesOptions, setIncTypesOptions] = useState();
 
-  const covidKeywords = ["covid19","covid","coronavirus"];
-
-  const covidException = (desc, keywords) => {
+  const checkCovidSpecialCase = (desc, keywords) => {
     const lowercasedDescription = desc.toLowerCase();
     return (
-      keywords.find((keyword) =>
-          lowercasedDescription.includes(keyword.toLowerCase()
-          )
+      keywords.find((keyword) => lowercasedDescription.includes(keyword.toLowerCase())
       ) || null
     );
   };
 
   const updateOptionsFromDescription = useCallback(
     async (description, options) => {
-       // specialcase to be handled for Covid keywords
+      // specialcase to be handled for Covid keywords
+      const covidKeywords = ["covid", "coronavirus"];
 
       if (options) {
-        if(covidException(description,covidKeywords)!= null){
-        
-          setIncTypesOptions(Object.values(options));
-          setIncidentTypePriAutocomplete({label: "COVID-19 Confirmed", 
-                                          value: "COVID-19 Confirmed", 
-                                          confidence: "1"});
-         
-          }
-        else{
-        const { updatedIncTypes, topIncType } = await getMultiPrediction(
-          description,
-          options
+        const covidOptionExist = options.filter((type) =>
+          type["label"].toLowerCase().includes("covid-19 confirmed")
         );
-        
-        setIncTypesOptions(Object.values(updatedIncTypes));
-        setIncidentTypePriAutocomplete(topIncType);
+
+        if (
+          checkCovidSpecialCase(description, covidKeywords) != null &&
+          covidOptionExist != null
+        ) {
+          const covidInc = covidOptionExist[0];
+          covidInc["confidence"] = "1.0";
+
+          setIncTypesOptions(Object.values(options));
+          setIncidentTypePriAutocomplete(covidInc);
```

Comment:
We should be able to object spread here. Change the last line to `setIncidentTypePriAutocomplete(...covidOptionIfExists, confidence: "1.0");`. Also double check if the confidence can just be passed in as a number - I think it'll be slightly cleaner.